src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
 import './App.css';
+import MyComponent from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] App.js component should be class-based, you'll see in the next steps why this is important. 🐱

Reply:
But the eslinters don't let me set class-based. Linters says that I should use functional components. What should I do?

Reply:
Gracias @vichuge siempre tan acertado 👍🏼